app/assets/javascripts/participants.js.coffee

```diff
@@ -60,3 +60,8 @@ $ ->
   $("#participants-list-table").bootstrapTable 'refresh', {silent: true}
   $("#participants-tracker-table").bootstrapTable 'refresh', {silent: true}
   $("#participant-info").bootstrapTable 'refresh', {silent: true}
+
+(exports ? this).visitSorter = (a, b) ->
```

Comment:
I disagree, as it is used in the context of sorting visits, I think visitSorter is just dandy